filament/backend/src/opengl/PlatformEGL.cpp

```diff
@@ -75,21 +75,22 @@ static void clearGlError() noexcept {
 
 // ---------------------------------------------------------------------------------------------
 
-PlatformEGL::PlatformEGL(EGLDisplay display) noexcept : 
-        mEGLDisplay(display) {
+PlatformEGL::PlatformEGL(EGLDisplay display) noexcept :
+        mEGLDisplay(display), mIsEGLDisplayExternal(display != EGL_NO_DISPLAY) {
 }
 
 Driver* PlatformEGL::createDriver(void* sharedContext) noexcept {
     if (mEGLDisplay == EGL_NO_DISPLAY) {
```

Comment:
nit: could have changed this to 
```suggestion
    if (!mIsEGLDisplayExternal) {
```

Reply:
fixed: https://github.com/shapr3d/filament/pull/80/commits/c63317f49f3b702a396b131116c06b347cf77dee